EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructure.png "[Message structure]")
+
+1. Expand the S_BGM segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it.   
+    The S_BGM segment in turn consists of subordinate segments and fields.
+
+    ![S_BGM segment](../../Assets/Screenshots/EDI/Operation/ExportDefBGM.png "[S_BGM segment]")
+
+2. Expand the *S_BGM C_C106* (Identification of a document/message by its number) segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it. In most cases it is required to specify this segment.   
+    The *S_BGM C_C106* segment has been expanded. The individual fields of this segment are displayed.
+
+    ![S_BGM C_C106 segment](../../Assets/Screenshots/EDI/Operation/ExportDefBGM_C106.png "[S_BGM  C_C106 segment]")
+
+3. Check the specification for the fields that are required to be filled. In most cases, you must fill the document number in the *D_1004* row (To identify a document). This is a numerical value that must be unique. For this reason, it is to be determined dynamically by a data source mapping. 
+
+4. Click the ![Add string attribute](../../Assets/Icons/Plus03.png "[Add string attribute]") button to the right of the *D_1004* row.   
+     - To the right of the *D_1004* row, only the ![Delete node](../../Assets/Icons/Trash10.png "[Delete node]")(Delete node) button is available now, with which you can remove the data, if required. The field name and description are no longer displayed in italics.  
+     - To the right of the *S_BGM C_C106*, the ![Add condition](../../Assets/Icons/AddCondition.png "[Add condition]") (Add condition) button is displayed with which you can define a condition for the output of this segment in the data mapping later.
+    - The *S_BGM C_C106* segment is now highlighted with a small black border. This indicates that the segment contains data.   
+    - The string attribute has been marked for a later data mapping.
+    
+5. Click the [SAVE] button to save the message structure.   
+    The *Exporter definitions* view is displayed. All defined string attributes are added to the attribute mapping function in the *Data sources* tab.
+
+6. If desired, check if the string attribute has been really marked for a later data mapping. To do this, open the message definition again and click the *Data sources* tab.  
```

Comment:
6-8 in einem separaten kleinen Procedure, z.B. Check the data source mapping? Es gibt mehrere Prozeduren hier, die sich darauf beziehen können.
Der Ausdruck "really marked" finde ich verwirrend. Warum würde es nicht passieren? Ist es nicht automatisch? "marked correctly"?

Reply:
Ne, es ist ja nur um zu zeigen, was passiert, wenn man bestimmte Dinge tut